packages/stencil/src/components/datacom-list/datacom-li.css

```diff
@@ -1,36 +1,30 @@
 :host {
   display: block;
+
+  --li-padding: 8px;
```

Comment:
Considering you're not reusing this value in multiple places in this file, you probably don't need to make this a variable. But there's no harm in doing so, so feel free to leave this if you want.